service/src/main/java/bio/terra/workspace/service/resource/controlled/ControlledResourceService.java

```diff
@@ -505,6 +507,13 @@ public String createAiNotebookInstance(
             "enablePet");
     jobBuilder.addParameter(ControlledResourceKeys.CREATE_NOTEBOOK_PARAMETERS, creationParameters);
     jobBuilder.addParameter(ControlledResourceKeys.NOTEBOOK_PET_SERVICE_ACCOUNT, petSaEmail);
+    jobBuilder.addParameter(
+        ControlledResourceKeys.CREATE_NOTEBOOK_MACHINE_TYPE, creationParameters.getMachineType());
+    jobBuilder.addParameter(
+        ControlledResourceKeys.CREATE_NOTEBOOK_ACCELERATOR_CONFIG,
+        AiNotebookApiConversions.fromApiAcceleratorConfig(
+            creationParameters.getAcceleratorConfig()));
+
```

Comment:
So, when updating these attributes in the database `UpdateNotebookResourceAttributesStep` (previously named `UpdateNotebookResourceLocationAttributesStep`) I wanted to avoid deserializing the API object. However, seeing as the other create steps use the API object regardless, I can remove these parameters.